python/paddle/fluid/contrib/slim/tests/test_mkldnn_int8_quantization_strategy.py

```diff
@@ -0,0 +1,211 @@
+                out = exe.run(inference_program,
+                              feed={
+                                  feed_target_names[0]: images,
+                                  feed_target_names[1]: labels
+                              },
+                              fetch_list=fetch_targets)
+                top1 += np.sum(out[1]) * len(data)
+                top5 += np.sum(out[2]) * len(data)
+                total_samples += len(data)
+            return top1 / total_samples, top5 / total_samples
+
+    def _warmup(self, reader=None, config_path=''):
+        com_pass = Compressor(
+            place=None,
+            scope=None,
+            train_program=None,
+            train_reader=None,
+            train_feed_list=[],
+            train_fetch_list=[],
+            eval_program=None,
+            eval_reader=reader,
+            eval_feed_list=[],
+            eval_fetch_list=[],
+            teacher_programs=[],
+            checkpoint_path='',
+            train_optimizer=None,
+            distiller_optimizer=None)
+        com_pass.config(config_path)
+        com_pass.run()
+
+    def test_compression(self):
+        if not fluid.core.is_compiled_with_mkldnn():
+            return
+
+        int8_model_path = test_case_args.int8_model_save_path
+        data_path = test_case_args.infer_data
+        fp32_model_path = test_case_args.infer_model
+        batch_size = test_case_args.batch_size
+
+        warmup_batch_size = test_case_args.warmup_batch_size
+        quantized_accuracy = test_case_args.quantized_accuracy
+
+        _logger.info(
+            'FP32 & INT8 prediction run: batch_size {0}, warmup batch size {1}.'.
+            format(batch_size, warmup_batch_size))
+
+        #warmup dataset, only use the first batch data
+        warmup_reader = paddle.batch(
+            self._reader_creator(data_path, False),
+            batch_size=warmup_batch_size)
+        config_path = self._update_config_file(fp32_model_path, int8_model_path)
+        self._warmup(warmup_reader, config_path)
+
+        _logger.info('--- FP32 prediction start ---')
+        val_reader = paddle.batch(
+            self._reader_creator(data_path, False), batch_size=batch_size)
+        fp32_model_result = self._predict(val_reader, fp32_model_path)
+        _logger.info('--- INT8 prediction start ---')
+        val_reader = paddle.batch(
+            self._reader_creator(data_path, False), batch_size=batch_size)
+        int8_model_result = self._predict(val_reader, int8_model_path)
+        _logger.info('--- comparing outputs ---')
+        _logger.info('Avg top1 INT8 accuracy: {0:.4}'.format(int8_model_result[
+            0]))
+        _logger.info('Avg top1 FP32 accuracy: {0:.4}'.format(fp32_model_result[
+            0]))
+        _logger.info('Accepted accuracy drop threshold: {0}'.format(
+            quantized_accuracy))
+        assert abs(int8_model_result[0] - fp32_model_result[

```

Comment:
Please, modify the assert condition so it fails only when
```
fp32_model_result[0] - int8_model_result[0] <= quantized_accuracy
```
It may happen that int8 accuracy is higher than fp32 accuracy, and that is OK.